src/util/RegistrationFormFactory.js

```diff
@@ -79,6 +79,9 @@ define([
     if (_.isString(regex)) {
       if (regex === '^[#/userName]') {
         var username = model.get('userName');
+        if (username == undefined) {
```

Comment:
use `===`

Reply:
it shall be sufficient to do `if (username)` 🔹

Reply:
It matters if both `userName` and `email` exists in the policy.

Reply:
theoretically we shall consider `null` which means `username == undefined` seems fine. but wonder we shall be more explicit like `isString and is not empty`?
anyway, minor thing.

Reply:
so whats the verdict? change to === or keep it unchanged?

Reply:
Right, we should consider `null` as well. Then the best is to do `model.has('userName')`